perfkitbenchmarker/linux_packages/ycsb.py

```diff
@@ -771,10 +769,9 @@ def Load(self, vms, workloads=None, load_kwargs=None):
     workloads = workloads or _GetWorkloadFileList()
     load_samples = []
     assert workloads, 'no workloads'
-    if FLAGS.ycsb_reload_database or not self.loaded:
```

Comment:
If the flag value for ycsb_reload_database is set to be false, will this change mean that the ycsb load is never triggered ?

Reply:
Correct.

The use case is if you want to run multiple tests against a DB that takes very long to load with different benchmark setups (number of machines, location of machines, etc.) reloading the DB everytime in this case is just not feasible.

Reply:
I discussed this internally, I think this will cause issues when the db is not loaded at the onset, and but no reload is required. Please support that behavior in your change.